zorg/buildbot/builders/BOLTBuilder.py

```diff
@@ -130,6 +130,16 @@ def getBOLTCmakeBuildFactory(
                 haltOnFailure=False,
                 flunkOnFailure=False,
                 env=env),
+            ShellCommand(
+                name='llvm-bolt-version-check',
+                command=(f"{boltNew} --version"),
+                description=('Check that llvm-bolt binary passes a simple test'
```

Comment:
The alternative would be some sort of dependency system in the test suite itself where you could say "must be able to print version" comes before "must be able to print help" comes before....and so on. This is not available in lit today, and is basically a build system unto itself. Fun idea, bad ROI.

So a smoke test like this is fine.